code/modules/clothing/shoes/magboots.dm

```diff
@@ -58,6 +58,14 @@
 	icon_state = "syndiemag0"
 	magboot_state = "syndiemag"
 
+/obj/item/clothing/shoes/magboots/commando
```

Comment:
Needs slowdown removed

Reply:
slowdown is already removed

Reply:
There is no slowdown, I've just tested it.
https://gyazo.com/4b5a5140ce3e52d60324440a11a67f20